packages/paste-website/src/component-examples/TimePickerExamples.ts

```diff
@@ -0,0 +1,166 @@
+export const RequiredTimePicker = `
+const RequiredTimePicker = (props) => {
+  const uidTP = useUID();
+  const uidHT = useUID();
+  return (
+    <>
+      <Label htmlFor={uidTP} required>
+        What time is your workday over?
+      </Label>
+      <TimePicker required id={uidTP} aria-describedby={uidHT} {...props} />
+      <HelpText id={uidHT}>Select a time.</HelpText>
+    </>
+  );
+};
+render(<RequiredTimePicker/>)
+`.trim();
+
+export const TimeRangePicker = `
+const TimeRangePicker = (props) => {
+  const [startTime, setStartTime] = React.useState('');
+  const startUidTP = useUID();
+  const endUidTP = useUID();
+  return (
+    <Stack orientation="horizontal" spacing="space80">
+      <Box>
+        <Label htmlFor={startUidTP}>Start time</Label>
+        <TimePicker id={startUidTP} onChange={(evt) => setStartTime(evt.target.value)} {...props} />
+      </Box>
+      <Box>
+        <Label htmlFor={endUidTP}>End time</Label>
+        <TimePicker id={endUidTP} min={startTime} {...props} />
```

Comment:
Since `min` doesn't affect time picker, I think we need to take it out, and the parts that relate to the startTime state, including the event handler. Great thing is this example will be oh so simple now!

Reply:
Good catch!! I'll change in the stories too